Release 18-11-24

Tschakki opened the pull request:
### Includes

- #235 
- #237 
- #239 
- #233 
- #245 
- #246

Tschakki commented on examples.md:
```diff
@@ -4,12 +4,399 @@
+
+### 6. Interact with the network
+
+Now that the network is started, let's try to send a `HelloTransaction` to our node to see if it gets accepted.
```
I am OK with `HelloTransaction`, as the new property is only called `hello`, too.
But if we want to change it to `HelloWorldTransaction`, we should also change it in the example code to avoid confusion: https://github.com/LiskHQ/lisk-sdk-examples/blob/development/hello_world/hello_transaction.js#L6

RachBLondon replied:
ah - sorry I have just seen this. Yes I think it will be easiest to go for hello world on everything, we can change this after the workshop if you would prefer.

Tschakki replied:
ok yes, I would prefer after workshop. When it's changed in the example app, we can update the docs, too :)

Tschakki commented on examples.md:
```diff
@@ -4,12 +4,399 @@
 > Check out the full code example for the [Hello World App on Github](https://github.com/LiskHQ/lisk-sdk-test-app/tree/development/hello_world).
 
-A simple App, showcasing a minimal setup of a blockchain application with 1 custom transaction type: the "Hello" transaction.
+Welcome to the step-by-step guide of creating the Hello World application with Lisk Alpha SDK.
```
What snippet you want to add?

RachBLondon replied:
@nazarhussain idea was to include the content of the file above in a snippet - we can pick this up again after the workshop tomorrow 👍

Tschakki commented on examples.md:
````diff
@@ -4,12 +4,399 @@
+
+Add the new transaction type to your application, by registering it to the application instance:
+
+```js
````
for `index.js` this should be the complete code

RachBLondon replied:
Yes, @nazarhussain said it would be simpler for this the easiest example to have both `index.js` and `hello_transaction.js` in one file

Tschakki replied:
I see, but if we merge them and then present the whole code on the tutorial page, this will be very big/long, so it might not necessarily help with getting an overview, imo. Maybe we can ask people during the workshop to get their opinion about this.

Tschakki commented on examples.md:
```diff
@@ -4,12 +4,399 @@
+
+#### TYP
+
+Set the HelloWorld transaction TYPE to 10. Every time a transaction is received, it gets differentiated by the type. The first 10 types, from 0-9 is reserved for the default Lisk Network functions.
```
@RachBLondon where is 0-99 mentioned?

Tschakki commented on examples.md:
````diff
@@ -4,12 +4,399 @@
+```js
+undoAsset(store) {
+    const sender = store.account.get(this.senderId);
+    const oldObj = { ...sender, asset: null };
````
@MaciejBaj solved this by adding a check in "applyAsset" func

RachBLondon commented on custom-transactions.md:
```diff
@@ -0,0 +1,95 @@
+The lifecycle of a transaction in general in Lisk SDK can be summarized as follows:
+
+1. A transaction is created and signed (off-chain). The script to do this is here `src/create_and_sign.ts`.
+2. The transaction is sent to a network. This can be done by a third party tool (like `curl` or `Postman`), but also using Lisk Commander, Lisk Hub or Mobile. All of the tools need to be authorized to access an HTTP API of a network node.
```
```suggestion
2. The transaction is sent to a network. This can be done by a third party tool (like `curl` or `Postman`), but also using Lisk Commander, Lisk Hub or Mobile. All of the tools need to be authorized to access an HTTP API of a network node.
```

Can we add a link to where we explain how to authorize?

RachBLondon commented on examples.md:
```diff
@@ -4,12 +4,400 @@
 
-A simple App, showcasing a minimal setup of a blockchain application with 1 custom transaction type: the "Hello" transaction.
+Welcome to the step-by-step guide of creating the Hello World application with Lisk Alpha SDK.
+A simple App, showcasing a minimal setup of a blockchain application with 1 [custom transaction](custom-transactions.md) type: the "Hello" transaction.
```
```suggestion
A simple App, showcasing a minimal setup of a blockchain application with 1 [custom transaction](custom-transactions.md) type: the "HelloWorld" transaction.
```

RachBLondon commented on examples.md:
```diff
@@ -4,12 +4,400 @@
+A simple App, showcasing a minimal setup of a blockchain application with 1 [custom transaction](custom-transactions.md) type: the "Hello" transaction.
 
+The purpose of Hello World application is to explain how to use and how to implement custom transaction with the Lisk SDK. 
+This custom transaction will extract the "hello" key value from the transaction asset property and save to the senders account.
```
```suggestion
This custom transaction will extract the "helloWorld" key value from the transaction asset property and save to the senders account.
```

RachBLondon commented on examples.md:
```diff
@@ -4,12 +4,400 @@
+### 3. Create a new transaction type
+
+For the Hello World App, we want to create a [custom transaction type](custom-transactions.md) `HelloWorld`: 
+If an account has enough balance to process `HelloWorld` transaction (fee is set to 1 LSK by default), the new "hello" property appears into this account's asset field.
```
```suggestion
If an account has enough balance to process `HelloWorld` transaction (fee is set to 1 LSK by default), the new "helloWorld" property appears into this account's asset field.
```

RachBLondon commented on examples.md:
```diff
@@ -4,12 +4,400 @@
+
+For the Hello World App, we want to create a [custom transaction type](custom-transactions.md) `HelloWorld`: 
+If an account has enough balance to process `HelloWorld` transaction (fee is set to 1 LSK by default), the new "hello" property appears into this account's asset field.
+So after sending a valid `{"type": 10, "senderId": "16313739661670634666L", ... "asset": { "hello": "world" } }` transaction, the sender's account changes from e.g.: `{ address: "16313739661670634666L", ..., asset: null }`, to `{ "address": "16313739661670634666L", ..., "asset": {"hello": "world"}} }`.
```
```suggestion
So after sending a valid `{"type": 10, "senderId": "16313739661670634666L", ... "asset": { "helloWorld": "a random string" } }` transaction, the sender's account changes from e.g.: `{ address: "16313739661670634666L", ..., asset: null }`, to `{ "address": "16313739661670634666L", ..., "asset": {"helloWorld": "a random string"}} }`.
```

RachBLondon commented on examples.md:
```diff
@@ -4,12 +4,400 @@
+If an account has enough balance to process `HelloWorld` transaction (fee is set to 1 LSK by default), the new "hello" property appears into this account's asset field.
+So after sending a valid `{"type": 10, "senderId": "16313739661670634666L", ... "asset": { "hello": "world" } }` transaction, the sender's account changes from e.g.: `{ address: "16313739661670634666L", ..., asset: null }`, to `{ "address": "16313739661670634666L", ..., "asset": {"hello": "world"}} }`.
+
+Now, let's create a new file `hello_transaction.js`, which is defining the new transaction type `HelloTransaction`:
```
```suggestion
Now, let's create a new file `hello_transaction.js`, which is defining the new transaction type `HelloWorldTransaction`:
```

RachBLondon commented on examples.md:
```diff
@@ -4,12 +4,400 @@
+
+}
+
+module.exports = HelloTransaction;
```
```suggestion
module.exports = HelloWorldTransaction;
```

RachBLondon commented on examples.md:
````diff
@@ -4,12 +4,400 @@
+module.exports = HelloTransaction;
+``` 
+
+> You find a version of [the complete `hello_transaction.js` file](https://github.com/LiskHQ/lisk-sdk-examples/blob/development/hello_world/hello_transaction.js) on Github
````
```suggestion
> You find a version of [the complete `hello_world_transaction.js` file](https://github.com/LiskHQ/lisk-sdk-examples/blob/development/hello_world/hello_world_transaction.js) on Github
```

RachBLondon commented on examples.md:
```diff
@@ -4,12 +4,400 @@
+
+#### prepare
+Prepares the necessary data for the `apply` and `undo` step.
+The "hello" property will be added only to sender's account, therefore it's the only resource needed in the `appluAsset` and `undoAsset` steps. 
```
```suggestion
The "hello" property will be added only to sender's account, therefore it's the only resource needed in the `applyAsset` and `undoAsset` steps. 
```

RachBLondon commented on examples.md:
```diff
@@ -4,12 +4,400 @@
+
+#### validateAsset
+Validation of the value of the "hello" property, defined by the HelloWorld transaction signer.
+The implementation below checks, that the value of the "hello" property needs to be a string, no longer than 64 characters. 
```
```suggestion
The implementation below checks, that the value of the "helloWorld" property needs to be a string, no longer than 64 characters. 
```

RachBLondon commented on examples.md:
```diff
@@ -4,12 +4,400 @@
+    if (!this.asset.hello || typeof this.asset.hello !== 'string' || this.asset.hello.length > 64) {
+        errors.push(
+            new TransactionError(
+                'Invalid "asset.hello" defined on transaction',
```
```suggestion
                'Invalid "asset.helloWorld" defined on transaction',
```

RachBLondon commented on examples.md:
````diff
@@ -4,12 +4,400 @@
+```js
+validateAsset() {
+    const errors = [];
+    if (!this.asset.hello || typeof this.asset.hello !== 'string' || this.asset.hello.length > 64) {
````
```suggestion
    if (!this.asset.helloWorld || typeof this.asset.helloWorld !== 'string' || this.asset.hello.length > 64) {
```

RachBLondon commented on examples.md:
```diff
@@ -4,12 +4,400 @@
+            new TransactionError(
+                'Invalid "asset.hello" defined on transaction',
+                this.id,
+                '.asset.hello',
```
```suggestion
                '.asset.helloWorld',
```

RachBLondon commented on examples.md:
```diff
@@ -4,12 +4,400 @@
+                'Invalid "asset.hello" defined on transaction',
+                this.id,
+                '.asset.hello',
+                this.asset.hello,
```
```suggestion
                this.asset.helloWorld,
```

RachBLondon commented on examples.md:
```diff
@@ -4,12 +4,400 @@
+
+That's where the custom logic of the Hello World app is implemented. 
+
+It shows how to store an additional information about accounts using the `asset` field. The content of property of "hello" transaction's asset gets saved into the "hello" property of the account's asset.
```
```suggestion
It shows how to store an additional information about accounts using the `asset` field. The content of property of "helloWorld" transaction's asset gets saved into the "hello" property of the account's asset.
```

RachBLondon commented on examples.md:
````diff
@@ -4,12 +4,400 @@
+```js
+applyAsset(store) {
+    const sender = store.account.get(this.senderId);
+    const newObj = { ...sender, asset: { hello: this.asset.hello } };
````
```suggestion
    const newObj = { ...sender, asset: { helloWorld: this.asset.helloWorld } };
```

RachBLondon commented on examples.md:
````diff
@@ -4,12 +4,400 @@
+}
+```
+#### undoAsset
+Inverse of `applyAsset`. Undoes the changes made in applyAsset step - reverts to the previous value of "hello" property, if not previously set this will be null.
````
```suggestion
Inverse of `applyAsset`. Undoes the changes made in applyAsset step - reverts to the previous value of "helloWorld" property, if not previously set this will be null.
```

RachBLondon commented on examples.md:
````diff
@@ -4,12 +4,400 @@
+```js
+//index.js
+const { Application, genesisBlockDevnet, configDevnet} = require('lisk-sdk'); // require application class, the default genesis block and the default config for the application
+const HelloTransaction = require('./hello_transaction'); // require the newly created transaction type 'HelloTransaction'
````
```suggestion
const HelloWorldTransaction = require('./hello_world_transaction'); // require the newly created transaction type 'HelloTransaction'
```

RachBLondon commented on examples.md:
```diff
@@ -4,12 +4,400 @@
+
+const app = new Application(genesisBlockDevnet, configDevnet); // create the application instance
+
+app.registerTransaction(HelloTransaction); // register the 'HelloTransaction' 
```
```suggestion
app.registerTransaction(HelloWorldTransaction); // register the 'HelloTransaction' 
```

RachBLondon commented on examples.md:
````diff
@@ -4,12 +4,400 @@
+For further interaction with the network, you can run the process in the background by executing:
+
+```bash
+npx pm2 start --name hello index.js # add the application to pm2 under the name 'hello'
````
```suggestion
npx pm2 start --name helloWorld index.js # add the application to pm2 under the name 'hello'
```

RachBLondon commented on examples.md:
```diff
@@ -4,12 +4,400 @@
+let h = createSendableTransaction(HelloTransaction, {
+	type: 10, // we want to send a transaction type 10 (= HelloTransaction)
+	asset: {
+		hello: 'world', // we save the string 'world' into the 'hello' asset
```
```suggestion
		helloWorld: 'test string', // we save the string 'test string' into the 'helloWorld' asset
```

RachBLondon commented on examples.md:
```diff
@@ -4,12 +4,400 @@
+
+For this, we utilize the API of the node and post the created transaction object to the transaction endpoint of the API.
+
+Because the API of every node is only accessible form localhost by default, you need to execute this query on the same server that your node is running on, unless you changed the config to make your API accessible to others or to the public.
```
```suggestion
Because the API of every node is only accessible from localhost by default, you need to execute this query on the same server that your node is running on, unless you changed the config to make your API accessible to others or to the public.
```

RachBLondon commented on examples.md:
````diff
@@ -4,12 +4,400 @@
+```js
+//index.js
+const { Application, genesisBlockDevnet, configDevnet} = require('lisk-sdk'); // require application class, the default genesis block and the default config for the application
+const HelloTransaction = require('./hello_transaction'); // require the newly created transaction type 'HelloTransaction'
````
```suggestion
const HelloWorldTransaction = require('./hello_world_transaction'); // require the newly created transaction type 'HelloWorldTransaction'
```

RachBLondon commented on examples.md:
```diff
@@ -19,4 +407,4 @@ It extends the pre-existing transaction type `TransferTransaction`.
 
 Here the Cashback transaction type pays out a 50% bonus reward to the sender of any Cashback transaction type.
```
```suggestion
Here the Cashback transaction type pays out a 10% bonus reward to the sender of any Cashback transaction type.
```
In the previous example it was 10%

RachBLondon commented on examples.md:
```diff
@@ -19,4 +407,4 @@ It extends the pre-existing transaction type `TransferTransaction`.
 Here the Cashback transaction type pays out a 50% bonus reward to the sender of any Cashback transaction type.
 
-So if Alice sends 100 token to Bob as a Cashback transaction, Bob would receive the 100 token and Alice would receive additional 50 tokens as a cashback.
+So e.g. if Alice sends 100 token to Bob as a Cashback transaction, Bob would receive the 100 token and Alice would receive additional 50 tokens as a cashback.
```
```suggestion
So e.g. if Alice sends 100 token to Bob as a Cashback transaction, Bob would receive the 100 token and Alice would receive additional 10 tokens as a cashback.
```

Tschakki commented:
has been merged in `dev-start-1.0.0` already

TalhaMaliktz committed:
Fix conflict main dev (#249)

Release 13-11-2024 (#244)

* [LISKDOCS-14] Update Docs UI to match the Lisk Rebranding (#235)

* Replace new fonts, update colors, etc.

* Apply changes to styling in light and dark mode. Update images

* Replace homepage images and style them

* Update logos, footer, images and buttons' styling

* Apply Rita Osokina's feedback

* Apply Dominic's feedback

* Apply Mona's feedback.

* Apply Mona's feedback.

* Update padding of 'heroBanner'

* [LISKDOCS-23] Update favicon (#237)

Update favicon

* [LISKDOCS-24]Update typography for mobile-view (#239)

Update typography for mobile-view

* Resolve conflicts in development (#241)

* Docs Release 28-06-2024 (#160)

* Add new section in docs

* Replace logos - update styling, configuration

* Create sidebar content, update config and test draft

* Groom docs and re-arrange content

Move Governance section to user docs
Style links
Add banners to light or dark mode

* Add hover styling

* Update intro.mdx

* Update intro.md

* Test no jekyll environment

* Update config

* Add Relay Bridge and Superbridge (#139)

* Add bridges

* Add and vice versa

* Update docs/lisk-tools/bridges.mdx



---------



* Update contracts.md

* Revamp style and structure of docs

* Update style

* Update links, style, text

* Update text and links at homepage

* Update index.js

* Update src/pages/index.js



* Update src/pages/index.js

* Add btns, style text, images, btns

* Refine landing page, styles and images

* Update index.js

* Update storage requirements (#148)

* Update intro pages and styles

* Update index.js

* Update dark-more images

* Add highlights to the user-docs intro page.

* Remove docs plugin, add multiple sidebars, update links, set landing page

* Add Tally links (#145)

* Add links

* Update docs/governance/overview.mdx



---------



* Delete userSidebar.js

* Update docusaurus.config.js

* Update docusaurus.config.js

* Update src/components/HomepageFeatures/index.js



* Update legacy chain service info (#149)

* Update legacy chain service info

* Add info about account initialization

* Apply suggestions from code review




* Apply suggestions from code review



* Remove image

* Create new section for Lisk L1 legacy docs

---------




* Add info about voting power calculation (#142)

* Add info about voting power calculation

* Add link

* Update link

* Apply feedback

* Apply feedback

* Apply feedback

* Apply feedback

* Apply feedback

* Add formulas for L1 fee (#138)

* Add formulas for l1 fee

* Apply feedback

* Add fee params

* Add gas tracker section

* Apply feedback

* Apply suggestions from code review



* Apply feedback

* Update docs/about-lisk/fees.md

* Apply feedback

---------



* Add coming soons (#159)

* Add coming soons

* Make bold

* Fix dev branch (#161)

* Release Docs 14-June-2024 (#151)

* Add new section in docs

* Replace logos - update styling, configuration

* Create sidebar content, update config and test draft

* Groom docs and re-arrange content

Move Governance section to user docs
Style links
Add banners to light or dark mode

* Add hover styling

* Update intro.mdx

* Update intro.md

* Test no jekyll environment

* Update config

* Add Relay Bridge and Superbridge (#139)

* Add bridges

* Add and vice versa

* Update docs/lisk-tools/bridges.mdx



---------



* Update contracts.md

* Revamp style and structure of docs

* Update style

* Update links, style, text

* Update text and links at homepage

* Update index.js

* Update src/pages/index.js



* Update src/pages/index.js

* Add btns, style text, images, btns

* Refine landing page, styles and images

* Update index.js

* Update storage requirements (#148)

* Update intro pages and styles

* Update index.js

* Update dark-more images

* Add highlights to the user-docs intro page.

* Remove docs plugin, add multiple sidebars, update links, set landing page

* Add Tally links (#145)

* Add links

* Update docs/governance/overview.mdx



---------



* Delete userSidebar.js

* Update docusaurus.config.js

* Update docusaurus.config.js

* Update src/components/HomepageFeatures/index.js



* Update legacy chain service info (#149)

* Update legacy chain service info

* Add info about account initialization

* Apply suggestions from code review




* Apply suggestions from code review



* Remove image

* Create new section for Lisk L1 legacy docs

---------




---------




* Remove line

---------




---------





* Release 12-11-2024 (#236)

[LISKDOCS-14] Update Docs UI to match the Lisk Rebranding (#235)

* Replace new fonts, update colors, etc.

* Apply changes to styling in light and dark mode. Update images

* Replace homepage images and style them

* Update logos, footer, images and buttons' styling

* Apply Rita Osokina's feedback

* Apply Dominic's feedback

* Apply Mona's feedback.

* Apply Mona's feedback.

* Update padding of 'heroBanner'

* Release 2: 12-11-2024 (#238)

* [LISKDOCS-14] Update Docs UI to match the Lisk Rebranding (#235)

* Replace new fonts, update colors, etc.

* Apply changes to styling in light and dark mode. Update images

* Replace homepage images and style them

* Update logos, footer, images and buttons' styling

* Apply Rita Osokina's feedback

* Apply Dominic's feedback

* Apply Mona's feedback.

* Apply Mona's feedback.

* Update padding of 'heroBanner'

* [LISKDOCS-23] Update favicon (#237)

Update favicon

---------





* Docs Release 28-06-2024 (#160)

* Add new section in docs

* Replace logos - update styling, configuration

* Create sidebar content, update config and test draft

* Groom docs and re-arrange content

Move Governance section to user docs
Style links
Add banners to light or dark mode

* Add hover styling

* Update intro.mdx

* Update intro.md

* Test no jekyll environment

* Update config

* Add Relay Bridge and Superbridge (#139)

* Add bridges

* Add and vice versa

* Update docs/lisk-tools/bridges.mdx



---------



* Update contracts.md

* Revamp style and structure of docs

* Update style

* Update links, style, text

* Update text and links at homepage

* Update index.js

* Update src/pages/index.js



* Update src/pages/index.js

* Add btns, style text, images, btns

* Refine landing page, styles and images

* Update index.js

* Update storage requirements (#148)

* Update intro pages and styles

* Update index.js

* Update dark-more images

* Add highlights to the user-docs intro page.

* Remove docs plugin, add multiple sidebars, update links, set landing page

* Add Tally links (#145)

* Add links

* Update docs/governance/overview.mdx



---------



* Delete userSidebar.js

* Update docusaurus.config.js

* Update docusaurus.config.js

* Update src/components/HomepageFeatures/index.js



* Update legacy chain service info (#149)

* Update legacy chain service info

* Add info about account initialization

* Apply suggestions from code review




* Apply suggestions from code review



* Remove image

* Create new section for Lisk L1 legacy docs

---------




* Add info about voting power calculation (#142)

* Add info about voting power calculation

* Add link

* Update link

* Apply feedback

* Apply feedback

* Apply feedback

* Apply feedback

* Apply feedback

* Add formulas for L1 fee (#138)

* Add formulas for l1 fee

* Apply feedback

* Add fee params

* Add gas tracker section

* Apply feedback

* Apply suggestions from code review



* Apply feedback

* Update docs/about-lisk/fees.md

* Apply feedback

---------



* Add coming soons (#159)

* Add coming soons

* Make bold

* Fix dev branch (#161)

* Release Docs 14-June-2024 (#151)

* Add new section in docs

* Replace logos - update styling, configuration

* Create sidebar content, update config and test draft

* Groom docs and re-arrange content

Move Governance section to user docs
Style links
Add banners to light or dark mode

* Add hover styling

* Update intro.mdx

* Update intro.md

* Test no jekyll environment

* Update config

* Add Relay Bridge and Superbridge (#139)

* Add bridges

* Add and vice versa

* Update docs/lisk-tools/bridges.mdx



---------



* Update contracts.md

* Revamp style and structure of docs

* Update style

* Update links, style, text

* Update text and links at homepage

* Update index.js

* Update src/pages/index.js



* Update src/pages/index.js

* Add btns, style text, images, btns

* Refine landing page, styles and images

* Update index.js

* Update storage requirements (#148)

* Update intro pages and styles

* Update index.js

* Update dark-more images

* Add highlights to the user-docs intro page.

* Remove docs plugin, add multiple sidebars, update links, set landing page

* Add Tally links (#145)

* Add links

* Update docs/governance/overview.mdx



---------



* Delete userSidebar.js

* Update docusaurus.config.js

* Update docusaurus.config.js

* Update src/components/HomepageFeatures/index.js



* Update legacy chain service info (#149)

* Update legacy chain service info

* Add info about account initialization

* Apply suggestions from code review




* Apply suggestions from code review



* Remove image

* Create new section for Lisk L1 legacy docs

---------




---------




* Remove line

---------




---------





* Remove comment and coming soon

* Resolve conflicts from main (#242)

* Docs Release 28-06-2024 (#160)

* Add new section in docs

* Replace logos - update styling, configuration

* Create sidebar content, update config and test draft

* Groom docs and re-arrange content

Move Governance section to user docs
Style links
Add banners to light or dark mode

* Add hover styling

* Update intro.mdx

* Update intro.md

* Test no jekyll environment

* Update config

* Add Relay Bridge and Superbridge (#139)

* Add bridges

* Add and vice versa

* Update docs/lisk-tools/bridges.mdx



---------



* Update contracts.md

* Revamp style and structure of docs

* Update style

* Update links, style, text

* Update text and links at homepage

* Update index.js

* Update src/pages/index.js



* Update src/pages/index.js

* Add btns, style text, images, btns

* Refine landing page, styles and images

* Update index.js

* Update storage requirements (#148)

* Update intro pages and styles

* Update index.js

* Update dark-more images

* Add highlights to the user-docs intro page.

* Remove docs plugin, add multiple sidebars, update links, set landing page

* Add Tally links (#145)

* Add links

* Update docs/governance/overview.mdx



---------



* Delete userSidebar.js

* Update docusaurus.config.js

* Update docusaurus.config.js

* Update src/components/HomepageFeatures/index.js



* Update legacy chain service info (#149)

* Update legacy chain service info

* Add info about account initialization

* Apply suggestions from code review




* Apply suggestions from code review



* Remove image

* Create new section for Lisk L1 legacy docs

---------




* Add info about voting power calculation (#142)

* Add info about voting power calculation

* Add link

* Update link

* Apply feedback

* Apply feedback

* Apply feedback

* Apply feedback

* Apply feedback

* Add formulas for L1 fee (#138)

* Add formulas for l1 fee

* Apply feedback

* Add fee params

* Add gas tracker section

* Apply feedback

* Apply suggestions from code review



* Apply feedback

* Update docs/about-lisk/fees.md

* Apply feedback

---------



* Add coming soons (#159)

* Add coming soons

* Make bold

* Fix dev branch (#161)

* Release Docs 14-June-2024 (#151)

* Add new section in docs

* Replace logos - update styling, configuration

* Create sidebar content, update config and test draft

* Groom docs and re-arrange content

Move Governance section to user docs
Style links
Add banners to light or dark mode

* Add hover styling

* Update intro.mdx

* Update intro.md

* Test no jekyll environment

* Update config

* Add Relay Bridge and Superbridge (#139)

* Add bridges

* Add and vice versa

* Update docs/lisk-tools/bridges.mdx



---------



* Update contracts.md

* Revamp style and structure of docs

* Update style

* Update links, style, text

* Update text and links at homepage

* Update index.js

* Update src/pages/index.js



* Update src/pages/index.js

* Add btns, style text, images, btns

* Refine landing page, styles and images

* Update index.js

* Update storage requirements (#148)

* Update intro pages and styles

* Update index.js

* Update dark-more images

* Add highlights to the user-docs intro page.

* Remove docs plugin, add multiple sidebars, update links, set landing page

* Add Tally links (#145)

* Add links

* Update docs/governance/overview.mdx



---------



* Delete userSidebar.js

* Update docusaurus.config.js

* Update docusaurus.config.js

* Update src/components/HomepageFeatures/index.js



* Update legacy chain service info (#149)

* Update legacy chain service info

* Add info about account initialization

* Apply suggestions from code review




* Apply suggestions from code review



* Remove image

* Create new section for Lisk L1 legacy docs

---------




---------




* Remove line

---------




---------





* Release 12-11-2024 (#236)

[LISKDOCS-14] Update Docs UI to match the Lisk Rebranding (#235)

* Replace new fonts, update colors, etc.

* Apply changes to styling in light and dark mode. Update images

* Replace homepage images and style them

* Update logos, footer, images and buttons' styling

* Apply Rita Osokina's feedback

* Apply Dominic's feedback

* Apply Mona's feedback.

* Apply Mona's feedback.

* Update padding of 'heroBanner'

* Release 2: 12-11-2024 (#238)

* [LISKDOCS-14] Update Docs UI to match the Lisk Rebranding (#235)

* Replace new fonts, update colors, etc.

* Apply changes to styling in light and dark mode. Update images

* Replace homepage images and style them

* Update logos, footer, images and buttons' styling

* Apply Rita Osokina's feedback

* Apply Dominic's feedback

* Apply Mona's feedback.

* Apply Mona's feedback.

* Update padding of 'heroBanner'

* [LISKDOCS-23] Update favicon (#237)

Update favicon

* Docs Release 28-06-2024 (#160)

* Add new section in docs

* Replace logos - update styling, configuration

* Create sidebar content, update config and test draft

* Groom docs and re-arrange content

Move Governance section to user docs
Style links
Add banners to light or dark mode

* Add hover styling

* Update intro.mdx

* Update intro.md

* Test no jekyll environment

* Update config

* Add Relay Bridge and Superbridge (#139)

* Add bridges

* Add and vice versa

* Update docs/lisk-tools/bridges.mdx



---------



* Update contracts.md

* Revamp style and structure of docs

* Update style

* Update links, style, text

* Update text and links at homepage

* Update index.js

* Update src/pages/index.js



* Update src/pages/index.js

* Add btns, style text, images, btns

* Refine landing page, styles and images

* Update index.js

* Update storage requirements (#148)

* Update intro pages and styles

* Update index.js

* Update dark-more images

* Add highlights to the user-docs intro page.

* Remove docs plugin, add multiple sidebars, update links, set landing page

* Add Tally links (#145)

* Add links

* Update docs/governance/overview.mdx



---------



* Delete userSidebar.js

* Update docusaurus.config.js

* Update docusaurus.config.js

* Update src/components/HomepageFeatures/index.js



* Update legacy chain service info (#149)

* Update legacy chain service info

* Add info about account initialization

* Apply suggestions from code review




* Apply suggestions from code review



* Remove image

* Create new section for Lisk L1 legacy docs

---------




* Add info about voting power calculation (#142)

* Add info about voting power calculation

* Add link

* Update link

* Apply feedback

* Apply feedback

* Apply feedback

* Apply feedback

* Apply feedback

* Add formulas for L1 fee (#138)

* Add formulas for l1 fee

* Apply feedback

* Add fee params

* Add gas tracker section

* Apply feedback

* Apply suggestions from code review



* Apply feedback

* Update docs/about-lisk/fees.md

* Apply feedback

---------



* Add coming soons (#159)

* Add coming soons

* Make bold

* Fix dev branch (#161)

* Release Docs 14-June-2024 (#151)

* Add new section in docs

* Replace logos - update styling, configuration

* Create sidebar content, update config and test draft

* Groom docs and re-arrange content

Move Governance section to user docs
Style links
Add banners to light or dark mode

* Add hover styling

* Update intro.mdx

* Update intro.md

* Test no jekyll environment

* Update config

* Add Relay Bridge and Superbridge (#139)

* Add bridges

* Add and vice versa

* Update docs/lisk-tools/bridges.mdx



---------



* Update contracts.md

* Revamp style and structure of docs

* Update style

* Update links, style, text

* Update text and links at homepage

* Update index.js

* Update src/pages/index.js



* Update src/pages/index.js

* Add btns, style text, images, btns

* Refine landing page, styles and images

* Update index.js

* Update storage requirements (#148)

* Update intro pages and styles

* Update index.js

* Update dark-more images

* Add highlights to the user-docs intro page.

* Remove docs plugin, add multiple sidebars, update links, set landing page

* Add Tally links (#145)

* Add links

* Update docs/governance/overview.mdx



---------



* Delete userSidebar.js

* Update docusaurus.config.js

* Update docusaurus.config.js

* Update src/components/HomepageFeatures/index.js



* Update legacy chain service info (#149)

* Update legacy chain service info

* Add info about account initialization

* Apply suggestions from code review




* Apply suggestions from code review



* Remove image

* Create new section for Lisk L1 legacy docs

---------




---------




* Remove line

---------




---------





* Remove comment and coming soon

---------





---------

Co-authored-by: Mona Bärenfänger <tschakki@posteo.de>
Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>
Co-authored-by: shuse2 <shus.toda@gmail.com>
